custom/icds_reports/utils/aggregation_helpers/distributed/child_health_monthly.py

```diff
@@ -98,25 +98,42 @@ def _state_aggregation_query(self, state_id):
             "(({} - child_health.opened_on::date)::integer >= 0) "
             "AND (child_health.closed = 0 OR (child_health.closed_on::date - {})::integer > 0)"
         ).format(end_month_string, start_month_string)
+        open_status_daily = (
+            "(child_health.opened_on::date <= {end_month_string}) "
+            "AND (child_health.closed = 0 OR child_health.closed_on::date > {end_month_string})"
+        ).format(end_month_string=end_month_string)
         alive_in_month = "(child_health.date_death IS NULL OR child_health.date_death - {} >= 0)".format(
             start_month_string
         )
+        alive_status_daily = "(child_health.date_death IS NULL OR child_health.date_death > {end_month_string})"\
+            .format(end_month_string=end_month_string)
         not_migrated = (
             "(agg_migration.is_migrated IS DISTINCT FROM 1 "
             "OR agg_migration.migration_date::date >= {start_month_string})"
         ).format(start_month_string=start_month_string)
+        not_migration_status_daily = (
+            "(agg_migration.is_migrated IS DISTINCT FROM 1)"
+        )
         registered = (
             "(agg_availing.is_registered IS DISTINCT FROM 0 "
             "OR agg_availing.registration_date::date >= {start_month_string})"
         ).format(start_month_string=start_month_string)
+        registered_month_end = (
```

Comment:
its daily, not month end

Reply:
@Rohit25negi / @challabeehyv The end of month status would be the status based on the status at month end?

Reply:
for older months, we check the status till month end

Reply:
That is automatic by the month filter in the below join. But as this variable is just storing  then the calculation of registration as of daily, it would be good to have it like that

Reply:
changed